charts/tensorleap/templates/rabbitmq-cm.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: rabbitmq-config
+data:
+  enabled_plugins: |
+    [rabbitmq_management].
+  rabbitmq.conf: |
+    log.file.level = debug
+    log.console = true
+    log.console.level = debug
+    queue_master_locator = min-masters
```

Comment:
What does this do?

Reply:
It selects a queue leader, but I just saw that it's optional

Reply:
so why aren't you removing this?

Reply:
Done